src/SprykerEco/Zed/Computop/Business/Hook/ComputopPostSaveHook.php

```diff
@@ -93,14 +100,12 @@ protected function setRedirect(TransferInterface $computopPaymentTransfer, Check
     /**
      * @param string $methodName
      *
-     * @throws \SprykerEco\Zed\Computop\Business\Exception\ComputopMethodMapperException
-     *
-     * @return \SprykerEco\Zed\Computop\Business\Hook\Mapper\Init\InitMapperInterface
+     * @return \SprykerEco\Zed\Computop\Business\Hook\Mapper\Init\InitMapperInterface|null
      */
     protected function getMethodMapper($methodName)
     {
         if (isset($this->methodMappers[$methodName]) === false) {
-            throw new ComputopMethodMapperException('The method mapper is not registered.');
+            return null;
```

Comment:
* This change looks like a BC break.

Reply:
Reverted

Reply:
There was bug but I fixed it with different method.